docs/concepts/internationalization.md

```diff
@@ -37,8 +34,29 @@ By default, `#getDescriptionId` will return `block.` or `item.` prepended to the
 The only purpose of a translation key is internationalization. Do not use them for logic. Use registry names instead.
 :::
 
-Localization methods
---------------------
+## Mod metadata
+
+Historically, A mod's name and description have been defined in [`mods.toml` files][modinfo] without translation support. Starting with Neoforge v20.4.179, translation files can override mod info using the following keys:
```

Comment:
I tweaked the wording of this paragraph slightly since the initial round of reviews.